html/captive-portal/lib/captiveportal/PacketFence/Controller/TLSProfile.pm

```diff
@@ -0,0 +1,180 @@
+package captiveportal::PacketFence::Controller::TLSProfile;
```

Comment:
It looks to me like you're importing the whole world in this module.

Please remove the modules you don't need